### PR TITLE
Adds an FRB Example

### PR DESCRIPTION
**Pull request by profxj**
And polishes things a bit further.

Adds CHANGES.rst too

### CODE REVIEW

File: astropath/bayesian.py [hunk 1]
```diff
@@ -2,48 +2,56 @@
...
             pchance_inverse :: Assign inverse to P_chance
             identical :: All the same
+        mag (float or np.ndarray):
+            Magnitudes of the sources;  assumed r-band
         half_light (float or np.ndarray, optional):
             Angular size of the galaxy in arcsec
             Only required for several methods
+        Pchance (float or np.ndarray, optional):
+            Chance probability
+            Required for linear and orig_inverse methods
```
Comment (KshitijAggarwal) [inline astropath/bayesian.py:31]: linear is gone now. So you can remove it from this statement. :)

File: astropath/bayesian.py [hunk 2]
```diff
@@ -147,12 +155,12 @@ def px_Oi(box_radius, frb_coord, eellipse, cand_coords,
...
         new_pa_gal = pa_gal + dtheta * units.deg
 
-        # p(w|M_i)
+        # p(w|O_i)
         # x, y gal
         x_gal = -r.value * np.sin(new_pa_gal).value
         y_gal = r.value * np.cos(new_pa_gal).value
         r_w = np.sqrt((xcoord-x_gal)**2 + (ycoord-y_gal)**2)
-        p_wMi = pw_Oi(r_w, theta_prior['r_half'][icand], theta_prior)
+        p_wMi = pw_Oi(r_w, theta_prior['ang_size'][icand], theta_prior)
```
Comment (KshitijAggarwal) [inline astropath/bayesian.py:163]: would be better to just rename p_wMi to p_w0i too.

Comment (KshitijAggarwal) [reply astropath/bayesian.py:163]: and the list p_wMis to p_w0is

File: astropath/bayesian.py [hunk 3]
```diff
@@ -2,48 +2,56 @@
...
 
+from astropath import chance
+
 from IPython import embed
 
 
-def raw_prior_Oi(Pchance, Sigma_m, method, half_light=None):
+def raw_prior_Oi(method, mag, half_light=None, Pchance=None):
     """
     Raw prior for a given set of Pchance values and/or n(m)
```
Comment (KshitijAggarwal) [inline astropath/bayesian.py:15]: Would be useful to mention that it uses Driver et al 2016 to estimate Sigma_m from the magnitude of the galaxy.

**Review by KshitijAggarwal — APPROVED** [review]

File: astropath/bayesian.py (hunk 1 above)
Comment (profxj) [reply astropath/bayesian.py:31]: removed

File: astropath/bayesian.py (hunk 2 above)
Comment (profxj) [reply astropath/bayesian.py]: done

File: astropath/bayesian.py (hunk 3 above)
Comment (profxj) [reply astropath/bayesian.py:15]: done

**Comment by profxj** [comment]
merging

**Pull request merged**